runtime/gc_glue_java/ArrayletObjectModel.cpp

```diff
@@ -43,6 +43,14 @@ GC_ArrayletObjectModel::AssertBadElementSize()
 	Assert_MM_unreachable();
 }
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+void
+GC_ArrayletObjectModel::AssertNotEmptyArrayletLeaves(UDATA sizeInElements, UDATA arrayletLeafCount)
+{
+	Assert_MM_true(0 == sizeInElements || arrayletLeafCount > 0);
```

Comment:
missing parentheses - the expression should be `(0 == sizeInElements) || (arrayletLeafCount > 0)`